tests/objects_test.py

```diff
@@ -91,16 +91,8 @@ def test_time(self):
     assert to_json(test_ts) == test_ts_json
     assert parse_json(test_ts_json) == test_ts
 
-    # We also support inputting datetime objects
-    dt = datetime.utcnow()
-    print to_json(dt)
-    print to_json(FaunaTime(dt))
-    assert to_json(dt) == to_json(FaunaTime(dt))
```

Comment:
`test_time_conversion` above is the right place. I think the tests there are sufficient.